assets/shaders/terrain.wgsl

```diff
@@ -14,8 +14,9 @@
 const TEXTURE_SIZE = 16.;
 const SHAPE_COLOR = vec4<f32>(1., 1., 1., 0.75);
 const SHAPE_THICKNESS = 0.15;
-// Keep thie array lenght in sync with /crates/terrain/src/shader.rs.
+// Keep these array lengths in sync with /crates/terrain/src/shader.rs.
 const MAX_KD_TREE_SIZE = 127u;
+const MAX_SQUARE_ARRAY_SIZE = 31u;
```

Comment:
Make sure this is small enough until we fix #216. Hopefully, once #216 is resolved, we will be able to remove this limitation and draw the selection boxes / circles around every single selected entity.